tests/test-cases/Group9-VIC-Admin/9-2-VICAdmin-CertAuth.robot

```diff
@@ -0,0 +1,52 @@
+*** Settings ***
+Documentation  Test 9-2 - VICAdmin CertAuth
+Resource  ../../resources/Util.robot
+Suite Setup  Install VIC Appliance To Test Server  certs=${true}
+Suite Teardown  Cleanup VIC Appliance On Test Server
+Default Tags
+
+*** Test Cases ***
+Wait
+    Run  sleep 30
+
+Display HTML
+    ${output}=  Run  curl -sk --cert /drone/src/github.com/vmware/vic/${vch-name}/cert.pem --key /drone/src/github.com/vmware/vic/${vch-name}/key.pem ${vic-admin}
+    Log To Console  ${output}
+    Should contain  ${output}  <title>VCH Admin</title>
+
+Get Portlayer Log
+     ${output}=  Run  curl -sk --cert /drone/src/github.com/vmware/vic/${vch-name}/cert.pem --key /drone/src/github.com/vmware/vic/${vch-name}/key.pem ${vic-admin}/logs/port-layer.log
+    Should contain  ${output}  Launching portlayer server
+
+Get VCH-Init Log
+    ${output}=  Run  curl -sk --cert /drone/src/github.com/vmware/vic/${vch-name}/cert.pem --key /drone/src/github.com/vmware/vic/${vch-name}/key.pem ${vic-admin}/logs/init.log
+    Should contain  ${output}  reaping child processes
+
+Get Docker Personality Log
+     ${output}=  Run  curl -sk --cert /drone/src/github.com/vmware/vic/${vch-name}/cert.pem --key /drone/src/github.com/vmware/vic/${vch-name}/key.pem ${vic-admin}/logs/docker-personality.log
+    Should contain  ${output}  docker personality
+
+# Get Container Logs
```

Comment:
It is run locally. That's very likely what's going on -- the docker errors were TLS errors: ```Warning: failed to get default registry endpoint from daemon (An error occurred trying to connect: Get https://192.168.218.165:2376/v1.21/info: x509: cannot validate certificate for 192.168.218.165 because it doesn't contain any IP SANs). Using system default: https://index.docker.io/v1/
An error occurred trying to connect: Post https://192.168.218.165:2376/v1.21/images/create?fromImage=busybox%3Alatest: x509: cannot validate certificate for 192.168.218.165 because it doesn't contain any IP SANs
```